_posts/2019-12-17-cancelling-hhvm-4.37.markdown

```diff
@@ -0,0 +1,20 @@
+---
+title: "Holiday schedule: cancelling HHVM 4.37"
+layout: post
+author: fred
+category: blog
+---
+
+We have decided to cancel next weeks' release of HHVM 4.37, and expect to delay
+4.38 until Thursday, the 2nd of January. HHVM 4.31 will be supported until the
+release of HHVM 4.38 or above. As HHVM 4.32 has long-term-support (LTS), it is
+unaffected by this change.
+
+Our usual schedule is to release on Mondays or Tuesdays, and we expect many
+people - both at Facebook, and among our users - to be on vacation for Tuesday
+and Wednesday for the next two weeks - both at Facebook, and among our users. We
```

Comment:
Has some redundancy with the phrase "both at Facebook, and among our users" since it's repeated twice.

```
Our usual schedule is to release on Mondays or Tuesdays, and we expect many
people, both at Facebook, and among our users, to be on vacation for Tuesday
and Wednesday for the next two weeks.
```